### PR TITLE
[documentation] migrate documentation to distroless

### PR DESCRIPTION
**Pull request by artbin**
## Description

Documentation module is based now on a distroless image.

## Why do we need it, and what problem does it solve?
We'd like to base our images on a distroless image.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries
```changes
section: documentation
type: feature
summary: documentation module is based on a distroless image.
impact_level: default
```

### CODE REVIEW

File: modules/810-documentation/images/web/werf.inc.yaml [hunk 1]
```diff
@@ -1,14 +1,7 @@
...
 image: {{ .ModuleName }}/{{ .ImageName }}
-from: {{ .Images.BASE_NGINX_ALPINE }}
-ansible:
-  setup:
-  - name: "Copy nginx.conf"
-    copy:
-      content: |
-        {{- .Files.Get (printf "%smodules/810-documentation/images/web/nginx.conf" .ModulePath) | nindent 8 }}
-      dest: /etc/nginx/nginx.conf
+from: common/nginx-static
```
Comment (RomanenkoDenys) [inline modules/810-documentation/images/web/werf.inc.yaml:4]:
```suggestion
fromImage: common/nginx-static
```

Comment (artbin) [reply modules/810-documentation/images/web/werf.inc.yaml]: done

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @artbin. I've started the workflow for label 'e2e/run/openstack'...

**Comment by z9r5** [comment]
/e2e/abort openstack;Standard;containerd;1.25 6807789913 6807789913-1-con-1-25 /sys/deckhouse-oss/install:pr6396

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'e2e/run/openstack'...

**Review by RomanenkoDenys — APPROVED** [review]
LGTM

**Pull request merged**